presto-kafka07/src/main/java/com/facebook/presto/kafka/KafkaConnectorConfig.java

```diff
@@ -76,7 +76,17 @@
     /**
      * Fetch size
      */
-    private int fetchSize = 10 * 1024 * 1024;
+    private DataSize fetchSize = new DataSize(10, Unit.MEGABYTE);
+
+    /**
+     * Default Query Interval
```

Comment:
Basing on the code logic, i.e. this value can be assigned. It is not a default value.